packages/jellyfish-api-core/__tests__/category/loan/placeAuctionBid.test.ts

```diff
@@ -208,31 +214,43 @@ async function setup (): Promise<void> {
   await alice.generate(1)
   await tGroup.waitForSync()
 
+  const tslaInterestPerBlock = new BigNumber(netInterest * tslaLoanAmount / (365 * blocksPerDay))
+  const currentHeight = await bob.rpc.blockchain.getBlockCount()
+  const tslaTotalInterest = tslaInterestPerBlock.multipliedBy(currentHeight - tslaTakeLoanHeight)
```

Comment:
change the height calculation after https://github.com/DeFiCh/jellyfish/pull/947/files#r784497139